README.md

```diff
@@ -66,12 +66,13 @@ You can also pass in options to custom messages:
 
 ```javascript
 Ember.get(this, 'flashMessages').add({
-  message      : 'I like alpacas',
-  type         : 'alpaca'
-  timeout      : 500,
-  priority     : 200,
-  sticky       : true,
-  showProgress : true
+  message            : 'I like alpacas',
+  type               : 'alpaca'
+  timeout            : 500,
+  priority           : 200,
+  sticky             : true,
+  showProgress       : true,
+  disappearTimeout : 500,
```

Comment:
So with the README changes, I'd prefer if you'd revert the whitespace changes, and update the formatting of L#75 to match the lines above it (aligned by `:`). As for naming, `disappearTimeout` makes sense, but maybe `extendedTimeout` would be more correct?